builder/build-image.sh

```diff
@@ -72,7 +72,7 @@ packer plugins install github.com/hashicorp/qemu
 packer plugins install github.com/hashicorp/ansible
 
 # Image tag is YYYY.MM.DD-short hash
-IMAGE_TAG="$(date "+%Y.%m.%d")-$(echo "$COMMIT_HASH" | cut -c 8)"
+IMAGE_TAG="$(date "+%Y.%m.%d")-$(echo "$COMMIT_SHA" | cut -c 8)"
```

Comment:
Should:
```
cut -c 8
```
be instead:
```
cut -c 1-8
```
?

Reply:
yes 🤦 sorry for that

Reply:
no worries, I should have spotted it in the review :D 

Let me know your thoughts about https://github.com/EGI-Federation/fedcloud-vmi-templates/issues/145 

Then I will make a PR with 1 or 2 changes.